docs/source/ko/_toctree.yml

```
@@ -52,8 +52,8 @@
       title: (번역 중) Hugging Face Hub API
     - local: in_translation
       title: (번역 중) Downloading files
-    - local: in_translation
-      title: (번역 중) Mixins & serialization methods
+    - local: package_reference/mixins
+      title: 미스인 & 직렬화 메서드
```

```suggestion
      title: 믹스인 & 직렬화 메서드
```

https://v2.ko.vuejs.org/v2/guide/mixins.html
vue.js 문서에서 mixins를 믹스인으로 번역하여서, 믹스인도 한번 고려해보면 좋을거 같습니다!

---

감사합니다. 오타였네요.